Userland/Libraries/LibWeb/HTML/Parser/HTMLTokenizer.cpp

```diff
@@ -1138,7 +1138,7 @@ Optional<HTMLToken> HTMLTokenizer::next_token()
                 }
                 ANYTHING_ELSE
                 {
-                    m_current_token.m_tag.attributes.append(HTMLToken::AttributeBuilder());
+                    m_current_token.m_tag.attributes.append({});
```

Comment:
```suggestion
                    m_current_token.m_tag.attributes.empend();
```

might work, I don't know if I made it work with aggregate initializers at any point in the past...

Reply:
How is that nicer? You really don't know intuitively what `empend()` does, I think. It would probably be even better to specify `append(HTMLToken::Attribute {})` IMO.

Reply:
Just that it does a direct initialization instead of a list-initialize + move-construct.
Ignore this if it doesn't work on the first try.

Reply:
This code is changed in a further commit, so it's basically irrelevant. :^)